src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ChangeLevelCalculatorTest.kt

```diff
@@ -60,7 +60,7 @@ internal class ChangeLevelCalculatorTest {
 
     @Test
     fun `should calculate Oasys Needs none`() {
-      val assessment = OffenderAssessment("12345", LocalDateTime.now(clock), null)
+      val assessment = OffenderAssessment("12345", LocalDateTime.now(clock), null, null)
```

Comment:
This looks a bit misleading - if the status is null then the assessment needs would not be retrieved as we would have ditched the assessments by this stage I think? I know it doesn't affect the test.

Reply:
I'm happy to change it, for me in unit tests I use `null` to say "I don't care about this value here" the filtering is done in another part of the code and this code just needs to accept whatever is passed in, we could use `"AnyStatus"` instead of null?